src/Microsoft.OData.Edm/Vocabularies/Values/EdmDateConstant.cs

```diff
@@ -4,30 +4,32 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.OData.Edm.Vocabularies
 {
     /// <summary>
     /// Represents an EDM date constant.
     /// </summary>
     public class EdmDateConstant : EdmValue, IEdmDateConstantExpression

```

Comment:
Would there be any value in naming this `EdmDateConstant` to `EdmDateOnlyConstant` or would it just result into unnecessary code churn? Note that the class is public.